### PR TITLE
chore: eventserver rename

### PR DESCRIPTION
**Pull request by MattLui**
Renamed "todos_change" to "db_changes" in event server and pushed a new image (1.0.1) to DH. Updated Listener.sql.

### CODE REVIEW

**Review by lukepow — APPROVED** [review]
Looks good!

File: eventserver/eventServer.js [hunk 1]
```diff
@@ -6,30 +6,28 @@ import createSubscriber from "pg-listen";
 const sseEmitter = new EventEmitter();
 
 const server = http.createServer((req, res) => {
-
   res.writeHead(200, {
     "Content-Type": "text/event-stream",
     "Cache-Control": "no-cache",
-    "Connection": "keep-alive",
+    Connection: "keep-alive",
```
Comment (timw-csi) [inline eventserver/eventServer.js:12]: very nit-picky, so advance apologies, but Connection is unquoted in the new file here. Still works the same of course, just breaks the consistency of headers appearance.

Comment (MattLui) [reply eventserver/eventServer.js:12]: Thanks, no need to apologize, love the detail-orientedness. That was a prettier-driven change. When I added back in quotes and pressed save, prettier reverted it back to no quotes. Must be conventional to have connection be unquoted in that context.

Comment (timw-csi) [reply eventserver/eventServer.js:12]: Oh I gotcha. Sounds like Prettier would unquote all of the object properties (at least the one word ones) if it had its druthers. Thanks for the cleanup work.
image

**Pull request merged**